### ISSUE TITLE
changing requirement.txt to torch==1.4.0 and torchvision==0.5.0 leads to pytorch version error

### ISSUE THREAD

**Issue by Ifrahmaqsood** [facebookresearch/mmf#767]
I am working in a conda environment
I have Cuda 10.0 installed on my tesla server and I installed torch==1.4.0+cu100 and torchvision==0.5.0+cu100 to make it compatible with cuda. Also, I changed the requirement.txt file with it and run 
**pip install --editable .**

I gave the following command to train the MCAN model giving my own feature path of **Grid Features**

**CUDA_VISIBLE_DEVICES=0 mmf_run config=projects/movie_mcan/configs/vqa2/defaults.yaml \model=movie_mcan \dataset=vqa2 \run_type=train**

After loading this

_2021-02-07T14:42:41 | mmf_cli.run: Namespace(config_override=None, local_rank=None, opts=['config=projects/movie_mcan/configs/vqa2/defaults.yaml', 'model=movie_mcan', 'dataset=vqa2', 'run_type=train'])
2021-02-07T14:42:41 | mmf_cli.run: Torch version: 1.4.0+cu100
2021-02-07T14:42:41 | mmf.utils.general: CUDA Device 0 is: Tesla K80
2021-02-07T14:42:41 | mmf_cli.run: Using seed 41955469
2021-02-07T14:42:41 | mmf.trainers.mmf_trainer: Loading datasets
2021-02-07T14:42:48 | torchtext.vocab: Loading vectors from /home/ifrahmaqsood/.cache/torch/mmf/glove.6B.300d.txt.pt_

It gives me an error

### _RuntimeError: version_ <= kMaxSupportedFileFormatVersion INTERNAL ASSERT FAILED at /pytorch/caffe2/serialize/inline_container.cc:132, please report a bug to PyTorch. Attempted to read a PyTorch file with version 3, but the maximum supported version for reading is 2. Your PyTorch installation may be too old. (init at /pytorch/caffe2/serialize/inline_container.cc:132)
frame #0: c10::Error::Error(c10::SourceLocation, std::string const&) + 0x33 (0x7fb7c7e88193 in /home/ifrahmaqsood/anaconda3/envs/mmf1/lib/python3.6/site-packages/torch/lib/libc10.so)
frame #1: caffe2::serialize::PyTorchStreamReader::init() + 0x1f5b (0x7fb7cafaeafb in /home/ifrahmaqsood/anaconda3/envs/mmf1/lib/python3.6/site-packages/torch/lib/libtorch.so)_

and the list goes on with frames

**python -m torch.utils.collect_env**

Collecting environment information...
PyTorch version: 1.4.0+cu100
Is debug build: No
CUDA used to build PyTorch: 10.0

OS: Ubuntu 16.04.7 LTS
GCC version: (Ubuntu 5.5.0-12ubuntu1~16.04) 5.5.0 20171010
CMake version: version 3.18.2

Python version: 3.6
Is CUDA available: Yes
CUDA runtime version: 10.0.130
GPU models and configuration:
GPU 0: Tesla K80
GPU 1: Tesla K80

Nvidia driver version: 430.64
cuDNN version: Probably one of the following:
/usr/lib/x86_64-linux-gnu/libcudnn.so.7.6.2
/usr/lib/x86_64-linux-gnu/libcudnn.so.8.0.2
/usr/lib/x86_64-linux-gnu/libcudnn_adv_infer.so.8.0.2
/usr/lib/x86_64-linux-gnu/libcudnn_adv_train.so.8.0.2
/usr/lib/x86_64-linux-gnu/libcudnn_cnn_infer.so.8.0.2
/usr/lib/x86_64-linux-gnu/libcudnn_cnn_train.so.8.0.2
/usr/lib/x86_64-linux-gnu/libcudnn_ops_infer.so.8.0.2
/usr/lib/x86_64-linux-gnu/libcudnn_ops_train.so.8.0.2

Versions of relevant libraries:
[pip3] numpy==1.19.5
[pip3] torch==1.4.0+cu100
[pip3] torchtext==0.5.0
[pip3] torchvision==0.5.0+cu100
[conda] torch                     1.4.0+cu100              pypi_0    pypi
[conda] torchtext                 0.5.0                    pypi_0    pypi
[conda] torchvision               0.5.0+cu100              pypi_0    pypi

Please if you could help me with this. As I cannot change the CUDA version due to shared server among multiple users.

**Comment by vedanuj** [comment]
Hi .. as the error says it seems to be related to your version of pytorch installed and when the code tries to load torchvision models the file version is not supported. I recommend upgrading your PT version.

**Comment by apsdehal** [comment]
If you can't update your cuda version, you can try installing PyTorch and torchvision from source in your conda environment. https://fb.workplace.com/groups/fair.nlp/permalink/3796903530428085

**Comment by Ifrahmaqsood** [comment]
thank you for your responses. I had to upgrade to cuda 10.1
